clusterctl/.gitignore

```diff
@@ -1 +1,3 @@
-clusterctl
\ No newline at end of file
+clusterctl
+*.yaml
```

Comment:
Would add this closer to the point of generation.

Reply:
Ended up removing it since those are excluded in examples/vsphere anyways.